conf/pf.conf.defaults

```diff
@@ -1093,6 +1093,11 @@ password=1234
 #
 # Shared KEY for vrrp protocol (Must be the same on all members).
 virtual_router_id=50
+#
+# active_active.centralized_deauth
+#
+# Centralize the deauthentication to the management node of the cluster.
+centralized_deauth=disabled
```

Comment:
I think the default should be enabled.
There are no downsides that I can see, whereas the reverse can fail in ways that are hard to debug for the uninitiated.